.github/workflows/e2e.yml

```
@@ -35,6 +35,7 @@ jobs:
           python3 manage.py migrate
           DJANGO_SUPERUSER_PASSWORD=testtest123 python manage.py createsuperuser --username admin --email test@test.com --no-input
           DJANGO_SUPERUSER_PASSWORD=testtest222 python manage.py createsuperuser --username admin2 --email test@test.com --no-input
+          echo -e "from django.contrib.auth.models import User\nuser=User.objects.create_user("user_test","user_test@user_test.com","123456")\nuser.set_password("123456")\nuser.save()\nexit()" > python manage.py shell
```

```suggestion
          echo -e "from django.contrib.auth.models import User\nuser=User.objects.create_user("user_test","user_test@user_test.com","123456")\nuser.set_password("123456")\nuser.save()\nexit()" | python manage.py shell
```
> @charlie0129 这个shell 貌似不能直接 echo 的呀

虽然我不知道 manage.py 这玩意能不能从 stdin 接收，你可能要加点参数. 但是你这个也没传给 stdin 啊,得用管道

---

哦 我傻了）